frontend/benefit/applicant/public/locales/fi/common.json

```diff
@@ -369,6 +369,7 @@
     "loginLabel": "Kirjaudu palveluun",
     "logoutLabel": "Kirjaudu ulos",
     "userAriaLabelPrefix": "Käyttäjä:",
+    "linkSkipToContent": "Hyppää pääsisältöön",
```

Comment:
Also, some other Helsinki projects seem to be using the phrasing "Siirry sivun pääsisältöön" and "Gå till huvudinnehåll".

Reply:
Just copied the texts from www.hel.fi 🤷  and didn't want to translate en/sv again...

Reply:
Ok, if it's there, then it's probably fine.